air/src/preparation_step/errors.rs

```diff
@@ -98,6 +100,9 @@ pub enum PreparationError {
     /// Failed to check peers' signatures.
     #[error(transparent)]
     DataSignatureCheckError(#[from] DataVerifierError),
+
+    #[error("invalid call result ID: {0}")]
+    InvalidCallResultId(ParseIntError),
```

Comment:
Could you elaborate in which way the call result is invalid? Its call-id is unexpected, e.g. too high ?

Reply:
Call IDs too high do not lead to `ParseIntError`.  But I should check if the message is clear to a user.